assets/test/SignPanel.test.ts

```diff
@@ -346,15 +371,82 @@ test.each([
         null,
       ),
     );
-    expect(wrapper.html()).toMatch('select');
-    (Object.keys(expected) as Array<SignModeOptions>).forEach((x) => {
-      expect(wrapper.find(`option[value="${x}"]`).exists()).toEqual(
-        expected[x],
-      );
+
+    expect(screen.getByRole('combobox')).toBeInTheDocument();
+
+    const options = screen
+      .getAllByRole('option')
+      .map((option) => option.getAttribute('value'));
+
+    (Object.keys(expected) as Array<SignModeOptions>).forEach((mode) => {
+      if (expected[mode] && expected[mode] === true) {
+        expect(options.includes(mode)).toBeTruthy();
+      } else {
+        expect(options.includes(mode)).toBeFalsy();
+      }
     });
   },
 );
 
+test('can enable/disable a sign', async () => {
+  const user = userEvent.setup();
+  const setConfig = jest.fn(() => true);
+
+  render(
+    React.createElement(
+      SignPanel,
+      { ...customModeSignProps(), setConfig },
+      null,
+    ),
+  );
+
+  await user.selectOptions(
+    screen.getByRole('combobox'),
+    screen.getByRole('option', { name: 'Off' }),
+  );
+  expect(setConfig).toHaveBeenCalledWith({ expires: null, mode: 'off' });
+
+  await user.selectOptions(
+    screen.getByRole('combobox'),
+    screen.getByRole('option', { name: 'Auto' }),
+  );
+  expect(setConfig).toHaveBeenCalledWith({ mode: 'auto' });
+});
+
+test('Disabling a sign clears any static text', async () => {
+  const user = userEvent.setup();
+  const setConfig = jest.fn(() => true);
+  render(
+    React.createElement(
+      SignPanel,
+      { ...customModeSignPropsWithStaticContent(), setConfig },
+      null,
+    ),
+  );
+
+  expect(
+    screen.getByAltText('Line one custom text input').getAttribute('value'),
+  ).toBe('foo');
+  expect(
+    screen.getByAltText('Line two custom text input').getAttribute('value'),
+  ).toBe('bar');
+
+  await user.selectOptions(
```

Comment:
These two tests were in `ViewerApp.test.tsx` but were just testing a single sign panel so I thought it made more sense to move them into this file